.github/workflows/docs.yml

```
@@ -0,0 +1,28 @@
+name: docs
+on:
+  push:
+    branches:
+      - main
```

```suggestion
      - main
  pull_request:
    branches:
      - main
```

so we can see the pipeline run in this PR, and in PRs that modify docs?

---

Oh, maybe that will update the live docs though?

---

Yea, it would, we'd need some staging location where PR-generated docs could go. It's pretty easy to just stand up a local version though:

```
pip install mkdocs-material
mkdocs serve
```

I'll add docs for this in a dev setup section in the readme file.